mule-extensions-api/src/main/java/org/mule/runtime/extension/api/ExtensionConstants.java

```diff
@@ -293,5 +293,20 @@
    */
   public static final String OBJECT_STORE_ELEMENT_NAMESPACE = "os";
 
+  /**
+   * The name of the parameter which configures pooling source item limits per poll.
+   *
+   * @since 1.4.0
+   */
+  public static final String POLLING_SOURCE_LIMIT_PARAMETER_NAME = "maxItemsPerPoll";
+
+  /**
+   * The description of the parameter which configures pooling source item limits per poll.
+   *
+   * @since 1.4.0
+   */
+  public static final String POLLING_SOURCE_LIMIT_PARAMETER_DESCRIPTION = "Limits the amount of items per poll that " +
+      "this polling source can send to the flow. ";
```

Comment:
I like this description a lot more :D I would also use it for the javadocs

Reply:
I think that the jdoc just needs to say that it is the description of that parameter.